media-video/ccextractor/ccextractor-0.85.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="A tool used to produce subtitles for TV recordings"
+DESCRIPTION="CCExtractor is a tool used to produce subtitles for TV \
+recordings from almost anywhere in the world."
+HOMEPAGE="https://www.ccextractor.org/"
+COPYRIGHT="2017 Carlos Fernandez and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/ccextractor/files/ccextractor/$portVersion/ccextractor-src-nowin.$portVersion.zip"
+CHECKSUM_SHA256="2ac21c6483e206a796d26d6adb7e969eb038a97ead9e2b2a7ee91b8b08c6882e"
+SOURCE_FILENAME="ccextractor-src-nowin.$portVersion.zip"
+SOURCE_DIR="ccextractor"
+PATCHES="ccextractor-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	ccextractor = $portVersion
+	cmd:ccextractor = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libcurl
+	lib:libiconv
+	lib:liblept
+	lib:libtesseract
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:find
+	cmd:g++
+	cmd:gcc
+	cmd:ld
+	devel:libcurl
+	devel:libiconv
+	devel:liblept
+	devel:libtesseract
+	"
+
+BUILD()
+{
+	cd linux
+	chmod +x ./*.sh
+	sh build
```

Comment:
the following is preferred as the default path could not contain the current dir: sh ./build